uber/tests/email/email_fixtures.py

```diff
@@ -0,0 +1,165 @@
+from uber.tests import *
+from unittest.mock import patch, Mock
+
+
+class EmailTestsConstants:
+    SUBJECT_TO_FIND = 'CoolCon9000: You Need To Know'
+    IDENT_TO_FIND = 'you_are_not_him'
+
+E = EmailTestsConstants
+
+
+@pytest.fixture
+def remove_all_email_categories(monkeypatch):
+    monkeypatch.setattr(AutomatedEmail, 'instances', OrderedDict())
+
+
+@pytest.fixture
+def add_test_email_categories(remove_all_email_categories):
```

Comment:
I'll have to give a closer look to how this fixture is used, but in general this fixture wouldn't be safe to call multiple times since if you try to instantiate an ``AutomatedEmail`` with an ident which has already been used, it'll error.

The canonical approach would be to make sure we clean up via a yield fixture, e.g.

```
@pytest.yield_fixture
def add_test_email_categories(remove_all_email_categories):
    yield AutomatedEmail(...)
    AutomatedEmail.instances.pop('you_are_not_him', None)
```